frontend/src/api/Assignment.ts

```diff
@@ -0,0 +1,11 @@
+import { createRequest } from '@/api/ApiProvider';
+import { HTTP_METHODS } from '@/helpers/authConstants';
+
+export const createAssignmentApi = async (data: FormData) => {
+  return createRequest({
+    endpoint: 'api/user/assignment',
+    method: HTTP_METHODS.post,
+    isMultipartHeaders: true,
+    data: data

```

Comment:
if a key and a value are the same you can omit one of them

Reply:
Done